api/app/routers/land_change/tree_cover_loss/tree_cover_loss.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import traceback
+import uuid
+from fastapi import APIRouter, BackgroundTasks, HTTPException, Request
+from fastapi import Response as FastAPIResponse
+from fastapi.responses import ORJSONResponse
+from app.models.land_change.tree_cover_loss import (
+    TreeCoverLossAnalyticsIn,
+    TreeCoverLossAnalyticsResponse,
+)
+from app.models.common.base import DataMartResourceLinkResponse, DataMartResourceLink
+from app.use_cases.analysis.tree_cover_loss.tree_cover_loss_service import (
+    TreeCoverLossService,
+)
+
+
+router = APIRouter(prefix="/tree_cover_loss")
+
+
+@router.post(
+    "/analytics",
+    response_class=ORJSONResponse,
+    response_model=DataMartResourceLinkResponse,
+    status_code=202,
+)
+def create(
+    *,
+    data: TreeCoverLossAnalyticsIn,
+    request: Request,
+    background_tasks: BackgroundTasks,
+):
+    try:
+        service = TreeCoverLossService(background_tasks)
+        service.do(data)
+
+        link = DataMartResourceLink(
+            link=f"{str(request.base_url).rstrip('/')}/v0/land_change/tree_cover_loss/analytics/{data.thumbprint()}"
+        )
```

Comment:
That's a big f-string. What do you think about this instead?
        link = DataMartResourceLink(
            link=(
                f"{str(request.base_url).rstrip('/')}"
                f"/v0/land_change/tree_cover_loss/analytics/{data.thumbprint()}"
            )
        )

Reply:
Another good catch. I found the _fastAPI_ way to do this for solid url creation: cda7728c7949405730ee4ce7e1a3f4f6b0aefe64